### PR TITLE
[INLONG-11157][Manager] Asynchronous processing agent installation

### LINKED ISSUE #11157: [Improve][Manager] Asynchronous processing agent installation
### Description

Asynchronous processing agent installation.

### InLong Component

InLong Manager

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by fuweng11**
Fixes #11157

### Motivation

Asynchronous processing agent installation.

### Modifications

Asynchronous processing agent installation.

### CODE REVIEW

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/cluster/InlongClusterServiceImpl.java [hunk 1]
```diff
@@ -126,6 +135,15 @@ public class InlongClusterServiceImpl implements InlongClusterService {
...
     private static final Gson GSON = new Gson();
+    private final ExecutorService executorService = new ThreadPoolExecutor(
+            5,
+            10,
+            10L,
+            TimeUnit.SECONDS,
+            new ArrayBlockingQueue<>(100),
+            new ThreadFactoryBuilder().setNameFormat("async-install-%s").build(),
+            new CallerRunsPolicy());
+    private final LinkedBlockingQueue<ClusterNodeRequest> installQueue = new LinkedBlockingQueue<>();
```
Comment (aloyszhang) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/cluster/InlongClusterServiceImpl.java:146]:
```suggestion
    private final LinkedBlockingQueue<ClusterNodeRequest> pendingInstallRequests = new LinkedBlockingQueue<>();
```

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/cluster/InlongClusterServiceImpl.java [hunk 2]
```diff
@@ -126,6 +135,15 @@ public class InlongClusterServiceImpl implements InlongClusterService {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(InlongClusterServiceImpl.class);
     private static final Gson GSON = new Gson();
+    private final ExecutorService executorService = new ThreadPoolExecutor(
+            5,
+            10,
+            10L,
+            TimeUnit.SECONDS,
+            new ArrayBlockingQueue<>(100),
+            new ThreadFactoryBuilder().setNameFormat("async-install-%s").build(),
```
Comment (aloyszhang) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/cluster/InlongClusterServiceImpl.java:144]:
```suggestion
            new ThreadFactoryBuilder().setNameFormat("agent-install-%s").build(),
```

Comment (fuweng11) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/cluster/InlongClusterServiceImpl.java]: Fixed.

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/cluster/InlongClusterServiceImpl.java (hunk 1 above)
Comment (fuweng11) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/cluster/InlongClusterServiceImpl.java]: Fixed.

**Review by aloyszhang — APPROVED** [review]
LGTM

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**